### PR TITLE
Set chefdk version on jenkins to 0.14.25

### PR DESCRIPTION
**Pull request by armiller**
@osuosl-cookbooks/chefs

### CODE REVIEW

**Comment by leel8on** [comment]
:+1:

File: spec/chef_ci_cookbook_template_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+describe 'osl-jenkins::chef_ci_cookbook_template' do
+  cached(:chef_run) do
+    ChefSpec::SoloRunner.new(CENTOS_6_OPTS).converge(described_recipe)
+  end
+
+  it 'should set chefdk version to 0.14.25' do
+    expect(chef_run.node['chef_dk']['version']).to eq('0.14.25')
+  end
+
+  it 'should install chefdk' do
```
Comment (ramereth) [inline spec/chef_ci_cookbook_template_spec.rb]: Mind removing the comment and just make this ``it do``?

File: spec/default_spec.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require_relative 'spec_helper'
+
+describe 'osl-jenkins::default' do
+  cached(:chef_run) do
+    ChefSpec::SoloRunner.new(CENTOS_7_OPTS).converge(described_recipe)
+  end
+  it 'converges successfully' do
+    expect { chef_run }.to_not raise_error
+  end
```
Comment (ramereth) [inline spec/default_spec.rb]: Do you intend to fix this test so that it actually runs or leave that for another PR? Looks like some stubbing is missing.

File: spec/chef_ci_cookbook_template_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+describe 'osl-jenkins::chef_ci_cookbook_template' do
+  cached(:chef_run) do
+    ChefSpec::SoloRunner.new(CENTOS_6_OPTS).converge(described_recipe)
+  end
```
Comment (ramereth) [inline spec/chef_ci_cookbook_template_spec.rb]: This should loop through C6 & C7.

**Comment by armiller** [comment]
@osuosl-cookbooks/chefs ready for final review

**Comment by ramereth** [comment]
:+1:

**Comment by armiller** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v0.7.25.  Have a nice day!
